app/components/project_submissions/item_component.rb

```diff
@@ -1,5 +1,7 @@
 module ProjectSubmissions
   class ItemComponent < ApplicationComponent
+    CURRENT_USER_SORT_CODE = 10_000_000 # current user's submission should always be first
```

Comment:
Bit of a workaround for now. After we get feature parity with the current submissions, we would be in a good place to revisit the design of submissions; and cleanly separate the users submission from the rest.